admin-portal/src/pages/needUpload/needUpload.tsx

```diff
@@ -77,13 +73,18 @@ export function NeedUpload() {
         <div className="uploadNeedsContainer">
           <form onSubmit={handleSubmit}>
             <p>Select the needs csv file that you want to upload.</p>
-            <input type="file" accept=".csv" onChange={handleChange} />
+            <input
+              type="file"
+              accept=".csv"
+              onChange={handleChange}
+              value={fileName}
+            />
             <button type="submit">Upload</button>
           </form>
         </div>
         <div className="error-list-div">
           {errorList.length > 0 &&
-            errorList.map((error: any) => <p>{error}</p>)}
+            errorList.map((error: string) => <p>{error}</p>)}
```

Comment:
```suggestion
            errorList.map((error: string) => <p key={error}>{error}</p>)} 
```
or some key if possible

Reply:
I don't think the `error` would be unique. We might need to create a unique key when we populate the array.